apollo-router/src/services/router/body.rs

```diff
@@ -1,133 +1,63 @@
-#![allow(deprecated)]
-use std::fmt::Debug;
-
+use axum::Error as AxumError;
 use bytes::Bytes;
-use futures::future::BoxFuture;
-use futures::FutureExt;
 use futures::Stream;
-use http_body::SizeHint;
-use hyper::body::HttpBody;
-use tower::BoxError;
-use tower::Service;
-
-pub struct RouterBody(super::Body);
-
-impl RouterBody {
-    pub fn empty() -> Self {
-        Self(super::Body::empty())
-    }
+use futures::StreamExt;
+use http_body::Frame;
+use http_body_util::combinators::UnsyncBoxBody;
+use http_body_util::BodyExt;
+use http_body_util::Empty;
+use http_body_util::Full;
+use http_body_util::StreamBody;
+use hyper::body::Body as HttpBody;
 
-    pub fn into_inner(self) -> super::Body {
-        self.0
-    }
+pub type RouterBody = UnsyncBoxBody<Bytes, AxumError>;
 
-    pub async fn to_bytes(self) -> Result<Bytes, hyper::Error> {
-        hyper::body::to_bytes(self.0).await
-    }
-
-    pub fn wrap_stream<S, O, E>(stream: S) -> RouterBody
-    where
-        S: Stream<Item = Result<O, E>> + Send + 'static,
-        O: Into<Bytes> + 'static,
-        E: Into<Box<dyn std::error::Error + Send + Sync>> + 'static,
-    {
-        Self(super::Body::wrap_stream(stream))
-    }
+pub(crate) async fn get_body_bytes<B: HttpBody>(body: B) -> Result<Bytes, B::Error> {
+    Ok(body.collect().await?.to_bytes())
 }
```

Comment:
should this be `into_bytes`?
```suggestion
pub(crate) async fn into_bytes<B: HttpBody>(body: B) -> Result<Bytes, B::Error> {
    Ok(body.collect().await?.to_bytes())
}
```

Reply:
done: 555606addd96cd8bb44af2fe00a44b0a08d2d671